### ISSUE TITLE
Expose each node's IODE accreditation status

### ISSUE THREAD

**Issue by jmckenna** [iodepo/odis-arch#401]
- "_The Management Group instructed ODIS to provide mechanisms to allow and encourage all ODIS partners to communicate their accreditation status (IODE or other)."_
- I imagine this has several implications
  - can this be automatically embedded in each node's ODISCat entry?
  - then ODIS can harvest ODISCat, and store the accreditation status in the ODIS graph
  - then the front-end search can access the accreditation status through the ODIS graph/Parquet/Solr output

Thoughts?

cc @pbuttigieg @Lucy-Scott

**Comment by pbuttigieg** [comment]
@jmckenna yes  - this should be recorded i  ODISCat. IODE Accreditation can be included by IODE itself in that scenario, so no verification would be needed.

**Comment by jmckenna** [comment]
Possible duplicate of #392
